packages/main/src/components/FilterBar/FilterBar.test.tsx

```diff
@@ -269,14 +269,14 @@ describe('FilterBar', () => {
       { attachTo: document.body.appendChild(document.createElement('div')) }
     );
 
-    const filterItemsFB = wrapper.find(FilterGroupItem);
-    const openFiltersDialogBtn = wrapper.find(Toolbar).find(Button).at(3);
+    const filterItemsFB = asFragment.find(FilterGroupItem);
+    const openFiltersDialogBtn = asFragment.find(Toolbar).find(Button).at(3);
     act(() => {
       openFiltersDialogBtn.prop('onClick')();
     });
-    wrapper.update();
+    asFragment.update();
 
-    const filterItemsDialog = wrapper.find('ui5-dialog').find(FilterGroupItem);
+    const filterItemsDialog = asFragment.find('ui5-dialog').find(FilterGroupItem);
```

Comment:
This whole thing can be removed and replaced with:
```js
    fireEvent.click(screen.getByText('Filters'));

    const [selectFilterBar, selectFilterDialog] = screen.getAllByText('SELECT');
    expect(selectFilterDialog).toHaveAttribute('required', 'true');
    expect(selectFilterDialog.parentNode.parentNode.parentNode.parentNode.querySelector('ui5-checkbox')).toHaveAttribute('disabled', 'true');
```

In the end, that test should look like:
```js
it('Group Filter Items mounted in Dialog', () => {
    render(
      // ... leave the render content as it is
    );

    fireEvent.click(screen.getByText('Filters'));

    const [selectFilterBar, selectFilterDialog] = screen.getAllByText('SELECT');
    expect(selectFilterDialog).toHaveAttribute('required', 'true');
    expect(selectFilterDialog.parentNode.parentNode.parentNode.parentNode.querySelector('ui5-checkbox')).toHaveAttribute('disabled', 'true');
  });
```